src/components/PeopleFilters.tsx

```diff
@@ -16,6 +51,10 @@ export const PeopleFilters = () => {
             type="search"
             className="input"
             placeholder="Search"
+            value={query}
+            onChange={(event) => {
+              createQuery(event.target.value);
+            }}
```

Comment:
move functions to variables and use variables in JSX. Please check your code and improve according this comment